DataFormats/CSCDigi/interface/CSCALCTDigi.h

```diff
@@ -104,6 +116,14 @@ class CSCALCTDigi {
   uint16_t bx_;
   uint16_t trknmb_;
   uint16_t fullbx_;
+  /// Run-3 introduces high-multiplicity bits for CSCs.
+  /// Note: In DN-20-016, 3 bits are allocated for HMT in the
+  /// ALCT board. These bits are copied into the ALCT digi in CMSSW
+  uint16_t hmt_;
+
+  enum class Version { Legacy = 0, Run3 };
+
+  Version version_;
```

Comment:
What is the default value of "version_", and where is this set?

Reply:
The default is now "Legacy" it appears.

Reply:
Uninitialized.

Reply:
@fwyzard @abrinke1 I set the version as ```Version::Legacy;``` by default in the ALCT/CLCT/LCT constructor.